src/App.tsx

```diff
@@ -1,24 +1,60 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import TodoList from './components/TodoList';
+import { Footer } from './components/Footer';
+import { GlobalContext } from './context/GlobalContext';
+import { FilterType } from './types/FilterTypes';
+import { Todo } from './types/Todo';
+import { ErrorNotification } from './components/ErrorNotification';
+import { ErrorEnum } from './types/ErrorEnum';
+import { client } from './utils/fetchClient';
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+export const USER_ID = 11325;
```

Comment:
for fix circular dependensy error move this constant in separate file (src/constans.ts) and import in each place where you need USER_ID